nuget/microsoft.cppwinrt.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>microsoft.cppwinrt</id>
+    <version>0.0.0</version>
```

Comment:
Probably what you want is more than one package (one per targeted SDK), whose version tracks the C++/WinRT compiler version